packages/server/graphql/mutations/helpers/createTeamAndLeader.ts

```diff
@@ -25,7 +25,14 @@ export default async function createTeamAndLeader(
   const {id: teamId, orgId} = newTeam
   const organization = await dataLoader.get('organizations').loadNonNull(orgId)
   const {tier, trialStartDate} = organization
-  const verifiedTeam = new Team({...newTeam, createdBy: userId, tier, trialStartDate})
+
+  const verifiedTeam = new Team({
+    ...newTeam,
+    createdBy: userId,
+    tier,
+    trialStartDate,
+    isPublic: true
```

Comment:
+1 This is called in `bootstrapNewUser`, do we want all onboarding teams to be public?

Reply:
Yep, all newly created teams should be public by default